packages/tempus/src/Tempus_StepperExplicitRK_impl.hpp

```diff
@@ -134,8 +135,12 @@ void StepperExplicitRK<Scalar>::setObserver(
   if (obs == Teuchos::null) {
     // Create default observer, otherwise keep current observer.
     if (stepperExplicitRKObserver_ == Teuchos::null) {
-      stepperExplicitRKObserver_ =
-        Teuchos::rcp(new StepperExplicitRKObserver<Scalar>());
+       if (stepperPL_->get<bool>("Use Embedded",false))
+          stepperExplicitRKObserver_ =
+             Teuchos::rcp(new StepperRKStartingStepSize<Scalar>());
+       else
+          stepperExplicitRKObserver_ =
+             Teuchos::rcp(new StepperExplicitRKObserver<Scalar>());
```

Comment:
I may be mistaken here but it seems like this optimal step size is only used if the observer is not set and only for embedded algorithm. Should we have more control here to set to use the optimal step size regardless of whether observers are set? Do users have the option to override this if they want to force their own step size instead while using the embedded algorithm?